HW03_Xingruo_Summer_Zhang.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "HW03_Xingruo_Summer_Zhang"
+author: "Summer Zhang"
+date: "7/20/2020"
+output: md_document
+---
+
+```{r}
+# Load Packages
+library(ggplot2)
+library(readr)
+library(magrittr)
+library(tidyr)
+library(dplyr)
+
+# Read Data
+us <- read_csv("us.csv")
+states <- read_csv("us-states.csv")
+```
+
+```{r}
+head(states)
+```
+
+## Graph 1: How Caes in the Entire U.S. Grew
+```{r}
+# Graph the cases
+ggplot(us, aes(x = date, y = cases, linetype = "Cases", color = "Cases")) +
+  geom_line(size = 1.5) +
+  # Add a layer of deaths accordingly
+  geom_line(aes(x = date, y = deaths, linetype = "Deaths", color = "Deaths"), size = 1.5) +
+  # solid for cases, dashed for deaths
+  scale_linetype_manual(values = c("solid", "dotted")) + 
```

Comment:
That's a great way to change lines and colors!

Reply:
Agreed! This is a very clever way to define line types and colors.